### DISCUSSION TITLE
ERROR @wdio/selenium-standalone-service: Error: Port 4444 is already in use.

### DISCUSSION THREAD

**Discussion by aliKhan94** [webdriverio/webdriverio#8322] [Q&A]
Hello everyone,

I am trying to run my webdriverio test in firefox browser using selenium-standalone service. For the first time it runs as expected. But when I rerun it give me below error

`Execution of 1 workers started at 2022-05-10T04:16:17.196Z

2022-05-10T04:16:17.290Z INFO @wdio/cli:launcher: Run onPrepare hook
2022-05-10T04:16:24.677Z ERROR @wdio/selenium-standalone-service: Error: Port 4444 is already in use.
    at Object.start (C:\Users\mdakhan1\Desktop\webdriverio_test\node_modules\selenium-standalone\lib\start.js:131:11)`

Note: error comes when I run 2nd time. But after killing the process on 4444, it works.

Please help me.

**Comment by christian-bromann** [comment]
@aliKhan94 can you provide some logs that help identify if a problem prevented the process from shutting down the selenium server?

**Reply by marcmyers** [reply]
Hi @christian-bromann I'm having the same issue after upgrading from 5 to 7.23.0 (I followed the tutorials provided and went from 5 > 6 > 7). I don't know howto export logs, but here is a screenshot of my logs set to trace level. 
![image](https://user-images.githubusercontent.com/5490442/186955770-1af69b94-ee0a-48ec-b373-02a9c0ec81a2.png)
Any help would be amazing as I've been trying to fix this for two days and may just have to go back to 5 since I did not have this issue when using that version of Webdriver.io

**Reply by marcmyers** [reply]
Also I can goto localhost:4444/ui/index.html#/ and clearly see it's still running: 
![image](https://user-images.githubusercontent.com/5490442/186968169-85407de4-4c60-448e-a9e3-12c410dd4679.png)
I have todo a taskkill to execute the tests a second time.

**Reply by spaudel1** [reply]
@aliKhan94 , sounds to me that this is somehow related to https://github.com/webdriverio/webdriverio/issues/8735. Please see the comments and check if its gonna be any help for you.
